lib/Doctrine/DBAL/Platforms/Keywords/PostgreSQL100Keywords.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Doctrine\DBAL\Platforms\Keywords;
+
+/**
+ * PostgreSQL 100 reserved keywords list.
+ *
+ * @link   www.doctrine-project.org
+ * @since  2.6
```

Comment:
As of https://github.com/doctrine/coding-standard/pull/9, `@since` will be not allowed. The `@link` above doesn't bring much value as well IMO.